src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpProxyCacheableRequestGroup.java

```diff
@@ -163,13 +163,14 @@ void onGroupResponseBegin(
         for (HttpCacheProxyCacheableRequest queuedRequest : queuedRequests)
         {
             final boolean notModified = etag != null && etag.equals(queuedRequest.ifNoneMatch);
+            final boolean requestLeader = groupRequest.request() == queuedRequest;
```

Comment:
I thought we eliminated the concept of "request leader" now that there is a request group concept.
This seems to be reintroducing the non-hierarchical approach from the old design.
What problem are you trying to solve?

Reply:
Yeah, it is just I couldn't find better name. How about `groupRequest`? When we create a cached response from the cacheable request I just want to know whether this request actually made to the backend or not so.

Reply:
The point I'm making is less about the name and more about the abstractions in place.
The old mental model was a proxy, where the front end would be aware it is talking to the backend, and the cache would get populated as a side-effect, much like the `tee` command.
The new mental model is hierarchical, where the lifecycle of the group request on the backend is now independent of the frontend request(s).
Passing this context to the frontend is breaking that abstraction and reintroducing awareness of the backend, much like the old mental model.